Kernel/style.m

```diff
@@ -170,7 +186,17 @@
   $ruleArrowStyle -> GrayLevel[0.65],
   $ruleGridColor -> GrayLevel[0.85],
   $ruleImageSizePerPlotRange -> 128,
+  
+  (* MultiwaySystem styles *)
+  
+  (* States graph *)
+  $statesGraphVertexStyle -> Directive[Opacity[0.7], Hue[0.62, 0.45, 0.87]],
+  $statesGraphEdgeStyle -> Directive[{Hue[0.75, 0, 0.35], Dashing[None]}],
+
+  (* Evolution causal graph *)
+  $evolutionCausalGraphEvolutionEdgeStyle -> Directive[{Hue[0.75, 0, 0.24], Dashing[None]}],
+  $evolutionCausalGraphCausalEdgeStyle -> Directive[{Hue[0.07, 0.78, 1], Dashing[None]}],
```

Comment:
@sskini2 Is it intentional that this is different from `$causalGraphEdgeStyle` (`EdgeStyle` for a non-multiway causal graph)?

Reply:
Yes, the design spec is different, as a means of making the causal edges more distinct from the evolution edges in the evolution causal graph (recall, causal edges used to be dashed here; SW then decided that they should be solid instead, but with a different color scheme).

Reply:
Yes, thanks for the clarification.